website/docs/r/is_lb.html.markdown

```diff
@@ -47,6 +47,8 @@ Review the argument references that you can specify for your resource.
 - `name` - (Required, String) The name of the VPC load balancer.
 - `profile` - (Required, Forces new resource, String) The profile to use for this load balancer. Supported value is `network-fixed`.
 - `resource_group` - (Optional, Forces new resource, String) The resource group where the load balancer to be created.
+- `route_mode` - (Optional, Forces new resource, Bool) Indicates whether route mode is enabled for this load balancer.
+  **NOTE** Currently, public load balancers are not supported with `route_mode` enabled.
```

Comment:
DO we need to specify 'profile' must be 'network-fixed' also here?

Reply:
refer this change for 'profile' doc change. https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3170/files. 'profile' doc update will go as part of this pr